test/integration/test-video-players.js

```diff
@@ -35,7 +35,7 @@ describe
   .run('amp-video', () => {
     runVideoPlayerIntegrationTests((fixture) => {
       const video = fixture.doc.createElement('amp-video');
-      video.setAttribute('src', '/examples/av/ForBiggerJoyrides.mp4');
+      video.setAttribute('src', '/examples/av/ForBiggerJoyrides-tiny.mp4');
```

Comment:
Should this be accompanied by a change to the `timeout` value used by these tests?

Reply:
That is a great point, I may need to experiment around those timeouts though.

Reply:
Removing the custom timeout value seems to be fine now.